Makefile

```diff
@@ -13,11 +13,5 @@ include common/common.mk
 upgrade-tests: $(VERSIONS)
 	test/run_upgrade_test 9.2:remote 9.4:local 9.5:local 9.6:local
 
-DG = /usr/bin/dg
-DG_EXEC = ${DG} --max-passes 25 --template Dockerfile.template --multispec specs/multispec.yml
-
-generate_dockerfiles:
-	for version in ${VERSIONS}; do \
-		${DG_EXEC} --distro centos-7-x86_64.yaml --multispec-selector version=$${version} --output $${version}/Dockerfile; \
-		${DG_EXEC} --distro rhel-7-x86_64.yaml --multispec-selector version=$${version} --output $${version}/Dockerfile.rhel7; \
-	done
+distgen:
+	VERSIONS="$(VERSIONS)" ./gen-dirs.sh
```

Comment:
Please use make targets instead of this script, and please develop them in `container-common-scripts` project ...

Running this script takes ~12s on my box, because distgen is pretty slow TBH:

    $ make distgen
    real    0m11.999s
    user    0m11.124s
    sys     0m0.827s

(at least at this point).  `make` should help with building it in parallel.

Reply:
I agree. Adding the rules to `container-common-scrtips` will make the distgen change much easier to adopt in other images if the maintainers choose to do so.